### DISCUSSION TITLE
Add colour for the UV index category in the header icon

### DISCUSSION THREAD

**Discussion by elydpg** [breezy-weather/breezy-weather#1118] [Ideas]
### Describe your idea

Similar to #738 for wind direction, we could add colour to indicate the UV index category in the header icon. It could look something like this:

![Screenshot_20240610-105514_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/8622264/bc8949f2-3f97-4194-b467-f1e40511eeb8)
![Screenshot_20240610-120744_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/8622264/586cb0b0-34c8-484a-9a67-a400b196d186)
![Screenshot_20240610-105514_Breezy Weather_1](https://github.com/breezy-weather/breezy-weather/assets/8622264/433a65f1-4c3a-49d2-bf3f-44d136065412)
![Screenshot_20240610-120744_Breezy Weather_1](https://github.com/breezy-weather/breezy-weather/assets/8622264/7f731dde-5777-4e9c-9256-02ddafb9d422)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
A new homepage is in progress (https://github.com/breezy-weather/breezy-weather/pull/1928), and the details on top will be removed in favor of Material 3 Expressive blocks.
The new UV block will include a colored dot, as wished.
